content/de/_index.html

```diff
@@ -12,38 +12,44 @@
+	</a>
+	
 	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/openbikesensor">
 		Github <i class="fab fa-github ml-2 "></i>
 	</a>
-  <a class="btn btn-lg btn-secondary mr-3 mb-4" href="{{< relref "/slack" >}}">
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="{{< relref "/slack" >}}">
 		Slack<i class="fab fa-slack ml-2 "></i>
 	</a>
 	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
 		Dokumentation <i class="fas fa-arrow-alt-circle-right ml-2"></i>
 	</a>
-	<p class="lead mt-5">Überholabstandsmessung für Radfahrende. Open source, open data, open science.</p>
+
 </div>
 {{< /blocks/cover >}}
 
 
 {{% blocks/section color="secondary" %}}
 <div class="container">
 
-Der OpenBikeSensor ist ein kleines technisches Gerät am Fahrrad, das während
-der Fahrt den Abstand von überholenden Autos misst.
+Wir sind eine Gruppe von aktiven Alltagsradler:innen und möchten, Radfahren sicherer und populärer machen. Überall! 
 
-Ein fertiges Produkt ist es noch nicht, man kann ihn nicht bei uns kaufen. Aber
-die Pläne und Infos sind alle öffentlich, schaut einfach im <a href="https://github.com/openbikesensor">Github</a>
-vorbei.  Dort liegen die Repositories der Firmware, des PCBs, der Gehäuse zum
-3D-Drucken, Front- und Backend des Daten-Portals und diese Dokuseite.
-Mitmachen ist dort natürlich auch möglich und gerne gesehen.
+Wie oft passiert es uns, dass uns ein Auto beinahe vom Sattel holt? Was wir alle kennen, überprüfen wir nun wissenschaftlich. Dazu haben wir einen Sensor entwickelt, der den Abstand misst und Überholmanöver Geodaten zuordnet. Wo ist es sicher? Wo nicht? Welche Uhrzeit? Welche Strecken? Der Sensor soll helfen, das zu erkennen und zu belegen.
 
-Die Community kommuniziert zum Großteil im offenen <a href="{{< relref "/slack" >}}">Slack</a>.
+{{< yt id=YrpipBDGe9s thumbnail="/thumbnails/YrpipBDGe9s.jpg" >}}
+	
+Der Überholabstandsmesser des OpenBikeSensor ist ein kleines technisches Gerät am Fahrrad. Während der Fahrt misst er den Abstand von überholenden Autos. Wir wünschen uns, dass er von vielen genutzt und weiterentwickelt wird. Die Anleitung zum Selberbauen und Montieren ist für alle als Open Source verfügbar <a href="https://github.com/openbikesensor" target="_blank" rel="noopener noreferrer nofollow">via Github</a>. 
 
-In das Datenportal können die Nutzer*innen die Daten ihres OpenBikeSensors
-hochladen und (falls sie wollen) ihre Daten als OpenData mit allen teilen.
+Unser Ziel ist, Schwachstellen und Verbesserungspotenzial in der Verkehrsinfrastruktur aufzuzeigen. Zudem wollen wir die gesellschaftliche Wirkung von Kampagnen wie beispielsweise #AnderthalbMeter (in Städten) messen. Daher ist es auch so wichtig, dass möglichst viele Alltagsradler:innen mit unseren Sensoren offene Daten (Open Data) erfassen. Mit den von allen gesammelten Daten wiederum lassen sich individuelle Auswertungen vornehmen; von Initiativen, Kommunen und von Euch (Open Citizen Science). 
```

Comment:
Gibt's nen Link zu `#AnderthalbMeter` damit das nicht so unerklärt da rumsteht?

Reply:
Wenn ein Link zur STVO bzw. deren Angaben zum Überholabstand gemeint ist: https://www.adfc.de/artikel/stvo-novelle-in-kraft-getreten

Reply:
> Kampagnen wie beispielsweise #AnderthalbMeter

Sorry, meine das. Wo ist diese Kampagne. Oder ist das ein Internet-Phänomen, dass dieses Hashtag einfach gerade im Kontext Überholabstand zu Radfahrenden verwendet wird?

Reply:
Das ist m.E. nur ein üblicher Hashtag zu dem Thema, den viele Radaktivist:innen nutzen. Mir ist kein zentrales Portal dazu bekannt.